src/modules/components/Header.tsx

```diff
@@ -0,0 +1,41 @@
+import { AppBar, SvgIcon, Toolbar, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import { theme } from 'styles/theme';
+import { ReactComponent as reactIcon } from '../../assets/icons/react-logo.svg';
+
+
+const Header: React.FC = () => {
+  return (
+    <Box sx={{display: 'flex', flexGrow: 1, alignItems: 'center' }}>
+      <AppBar sx={{
+        backgroundColor: theme.palette.primary.contrastText,
+        color: theme.palette.primary.dark,
+        paddingLeft: 2,
+        paddingRight: 2,
+        boxShadow: 'none',
+        borderBottom: '1px solid #E5EDF2',
```

Comment:
Color should be a variable in a separate file. Or use it from the theme as you did above